matrix-api.h

```diff
@@ -110,8 +110,15 @@ void matrix_api_bad_response(MatrixConnectionData *ma, gpointer user_data,
         int http_response_code, struct _JsonNode *json_root);
 
 
-
-
+/**
+ * Get roomid by alias
```

Comment:
Also, I'd suggest you put it among the other functions which hit a specific endpoint. Why not put it after `matrix_api_sync`, which would be consistent with where you have put it in the C?